source/_integrations/velux.markdown

```diff
@@ -67,6 +67,31 @@ automation:
     - service: velux.reboot_gateway
 ```
 
+## Attributes
+
+Cover entity with integrated rain sensor has extra attributes to represent the state of the limitation sensor.
+
+| Name | Description |
+| ---- | ----------- |
+| `limitation_min` | Min limitation of window. `limitation_min` => 0 no rain; `limitation_min` => 93 the window can only open for 7 % , so it means, its raining
```

Comment:
Really I do not know. 
I have only one Window Type to test this. I haven't found any documentation on whether there are different values for different windows.

The code run on my HA instance since 1 Month I had never seen other values.

@pergolafabio has also test the code. Did you seen ever other values?

Reply:
no, never seen other values, always 93 or 0 , still running here

Reply:
If this is the case, should the attribute just be "limited: True/False" or "raining: True/False"?

If we don't want to make it binary, the value should be translated to better map to the home assistant cover entity model where home assistant 0 means fully closed and 100 means fully opened. What's called `limitation_min` here with value 0 or 93 should instead be 100 or 7, and be called something like max_open?